docs/data/data-grid/editing/ValidateRowModelControlGrid.tsx

```diff
@@ -1,12 +1,13 @@
 import * as React from 'react';
 import Box from '@mui/material/Box';
-import { DataGrid, GridColumns, GridRowsProp } from '@mui/x-data-grid';
 import {
-  randomCreatedDate,
-  randomEmail,
-  randomTraderName,
-  randomUpdatedDate,
-} from '@mui/x-data-grid-generator';
+  DataGrid,
+  GridColumns,
+  GridPreProcessEditCellProps,
+} from '@mui/x-data-grid';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const VISIBLE_FIELDS = ['name', 'email', 'dateCreated', 'lastLogin'];
```

Comment:
The last column is not visible.

```suggestion
const VISIBLE_FIELDS = ['name', 'email', 'dateCreated', 'lastUpdated'];
```

Reply:
I realized I was looking at the wrong example in the doc 🙈